packages/web-client/app/components/card-pay/deposit-workflow/transaction-status/index.css

```diff
@@ -0,0 +1,47 @@
+.transaction-status {
+  padding: var(--boxel-sp-xxl) var(--boxel-sp-lg);
+}
+
+.transaction-status > * + * {
+  margin-top: var(--boxel-sp-xxl);
+}
+
+.transaction-status__description {
+  margin: 0;
+  font-size: 1.125rem; /* 18px */
+  font-weight: 700;
+  line-height: calc(24/18);
+}
+
+.transaction-status__steps {
+  color: var(--boxel-dark);
+  font: 600 var(--boxel-font);
+  letter-spacing: var(--boxel-lsp-xs);
+}
+
+.transaction-status__steps > li > span {
```

Comment:
Made a task in the backlog to update Boxel to have more flexible height for progress steps: 

https://linear.app/cardstack/issue/CS-746/progress-steps-height-should-adjust-to-its-content